doc/releases/changelog-dev.md

```diff
@@ -460,6 +460,12 @@
 * File `qcut.py` in `qml.transforms` reorganized into multiple files in `qml.transforms.qcut`
   [3413](https://github.com/PennyLaneAI/pennylane/pull/3413)
 
+* Replace `QuantumTape` with `QuantumScript` and/or `AnnotatedQueue` in all tests.
```

Comment:
We usually do not add entries for changing the tests, the next entry for Operation recorder is useful though.

Reply:
i thought maybe because of the size it was merited, but fair enough!